applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -268,7 +413,7 @@ static void stream_qos_set_cb(struct bt_audio_stream *stream)
 
 	LOG_DBG("Stream QOS set: %p", stream);
 
-	if (playing_state) {
+	if (ep_state_check(stream->ep, BT_AUDIO_EP_STATE_QOS_CONFIGURED)) {
```

Comment:
This may be correct, but I would like to discuss.

Reply:
Code rewrite has updated this so no longer relevant.